### PR TITLE
[ADD] Sales: new page on eWallets and gift cards programs

### PR DESCRIPTION
**Pull request by toaa-odoo**
Adding a new page on eWallets and gift cards programs for the Sales app documentation.

Taskid: 3086636

### CODE REVIEW

**Comment by toaa-odoo** [comment]
Hi @meng-odoo ! Here is the page I made on eWallets and gift cards for the Sales app.  I intentionally left out the eCommerce and PoS integration since I believe these should be documented by their respective marketing experts in their own doc section :).

I was not sure about the toctree placement, so I just put it there. Feel free to change it! Let me know if any changes are required :)

**Review by meng-odoo — APPROVED** [review]
Hi @toaa-odoo! I made small language changes throughout the doc, for clarity/consistency and to remove the use of second-person point of view ("you"). I also changed some of the images to make them tablet-friendly, and to show more of the page. Please let me know if you have any questions or comments, and thanks again for writing this! :)

I'm approving and tagging @mivu-odoo for final review.

File: content/applications/sales/sales/products_prices/ewallets_giftcards.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+========
+
+eWallets allow customers to save credits on their online account and use these credits as a payment
+method when buying items in an online store or a brick-and-mortar store. eWallets can also be used
+to centralize multiple :ref:`gift cards <ewallet_gift/gift-cards>`.
+
+Before creating an eWallet program, it is necessary to create an eWallet **top-up** product.
+Top-ups are pre-defined digital credit values added to an eWallet in exchange for its equivalent in
+real currency. These credits can then be used as a payment method in the eCommerce shop or
+:abbr:`PoS (Point of Sale)`. Top-up values can be of different amounts.
```
Comment (StraubCreative) [inline content/applications/sales/sales/products_prices/ewallets_giftcards.rst:22]: Can break a little later on line 19
```suggestion
Before creating an eWallet program, it is necessary to create an eWallet **top-up** product. Top-ups
are pre-defined digital credit values added to an eWallet in exchange for its equivalent in real
currency. These credits can then be used as a payment method in the eCommerce shop or :abbr:`PoS
(Point of Sale)`. Top-up values can be of different amounts.
```

File: content/applications/sales/sales/products_prices/ewallets_giftcards.rst [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  :guilabel:`Create`.
+- :guilabel:`Currency`: select the currency to use for the eWallet program
+- :guilabel:`Company`: select the company for which the program is valid and available
+- :guilabel:`Available On`: select the applications on which the program is valid and available
+- :guilabel:`Website`: select the website on which the program is valid and available. Leave this
+  field empty to include all websites.
+- :guilabel:`Point of Sale`: select the :abbr:`PoS (Point of Sale)` in which the program is valid
+  and available. Leave this field empty to include all :abbr:`PoS (Point of Sale)`.
+
+.. image:: ewallets_giftcards/ewallet-configuration.png
```
Comment (StraubCreative) [inline content/applications/sales/sales/products_prices/ewallets_giftcards.rst:59]: [Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~3x.

File: content/applications/sales/sales/products_prices/ewallets_giftcards.rst [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+Once the program is configured, click the :guilabel:`Generate eWallet` button in the upper-left
+corner to generate eWallets. eWallets can be generated based on :guilabel:`Customers` and/or
+:guilabel:`Customer Tags`. The quantity is automatically adapted according to the
+:guilabel:`Customers` and :guilabel:`Customer Tags` selected. Then, set the :guilabel:`eWallet
+value`. Finally, set the :guilabel:`Valid Until` period if applicable.
+
+Generated eWallets can be accessed through the :guilabel:`eWallets` smart button in the upper-right
+corner. From there, :guilabel:`Send` or :guilabel:`Share` the eWallets via email or a URL link.
+
+.. image:: ewallets_giftcards/ewallet-share.png
```
Comment (StraubCreative) [inline content/applications/sales/sales/products_prices/ewallets_giftcards.rst:72]: [Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~2x.

File: content/applications/sales/sales/products_prices/ewallets_giftcards.rst [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+- :guilabel:`Print Report`: select :guilabel:`Gift Card`
+- :guilabel:`Currency`: select the currency to use for the gift card program
+- :guilabel:`Company`: select the company for which the program is valid and available
+- :guilabel:`Available On`: select the applications on which the program is valid and available
+- :guilabel:`Website`: select the website on which the program is valid and available. Leave this
+  field empty to include all websites.
+- :guilabel:`Point of Sale`: select the :abbr:`PoS (Point of Sale)` in which the program is valid
+  and available. Leave this field empty to include all :abbr:`PoS (Point of Sale)`.
+
+.. image:: ewallets_giftcards/giftcard-configuration.png
```
Comment (StraubCreative) [inline content/applications/sales/sales/products_prices/ewallets_giftcards.rst:122]: [Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~3x.

File: content/applications/sales/sales/products_prices/ewallets_giftcards.rst [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+Customers` or :guilabel:`Selected Customers`. Set the :guilabel:`Quantity to generate` for
+:guilabel:`Anonymous Customers`, or select the :guilabel:`Customers` and/or :guilabel:`Customer
+Tags` for :guilabel:`Selected Customers`. Then, set the :guilabel:`Gift Card value`. Finally, set
+the :guilabel:`Valid Until` period if applicable.
+
+Generated gift cards can be accessed through the :guilabel:`Gift Cards` smart button in the
+upper-right corner. From there, :guilabel:`Send` or :guilabel:`Share` the gift cards via email or a
+URL link.
+
+.. image:: ewallets_giftcards/giftcard-share.png
```
Comment (StraubCreative) [inline content/applications/sales/sales/products_prices/ewallets_giftcards.rst:137]: [Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~2x.

**Review by StraubCreative — APPROVED** [review]
Hi @toaa-odoo 👋 

Nice doc!

I just had a couple of suggestions which I'll quickly address in a following commit before passing on to DR, in order to save us some time:
- line break adjustment on line 19
- compress screenshot images
- remove unused screenshots from this file's image directory

Great work y'all 🚀 
cc: @meng-odoo

**Comment by StraubCreative** [comment]
Final review change requests addressed on [4a57947](https://github.com/odoo/documentation/pull/3241/commits/4a579477e1f7308482a68ddc01664a137f8e297e). All checks pass 🚀 

@odoo/doc-review this new doc on eWallet and gift card programs is ready for your review 🙂

File: content/applications/sales/sales/products_prices.rst [hunk 6]
```diff
@@ -11,3 +11,4 @@ Products & Prices
    products_prices/taxes
    products_prices/prices
    products_prices/returns.rst
+   products_prices/ewallets_giftcards.rst
```
Comment (AntoineVDV) [inline content/applications/sales/sales/products_prices.rst:14]:
```suggestion
   products_prices/returns
   products_prices/ewallets_giftcards
```

File: content/applications/sales/sales/products_prices/ewallets_giftcards.rst [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+   :align: center
+   :alt: eWallets send and share buttons
+
+Click on an eWallet to change the :guilabel:`Expiration Date`, :guilabel:`Partner`, or
+:guilabel:`Balance`. The :guilabel:`Code` of an eWallet *cannot* be changed, deleted, or duplicated.
+
+Gift cards
+==========
+
+.. _ewallet_gift/gift-cards:
```
Comment (AntoineVDV) [inline content/applications/sales/sales/products_prices/ewallets_giftcards.rst:82]:
```suggestion
.. _ewallet_gift/gift-cards:

Gift cards
==========
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**